dkg/dkg.go

```diff
@@ -181,8 +192,60 @@ func setupP2P(ctx context.Context, datadir string, p2pConf p2p.Config, peers []p
 	}, nil
 }
 
-func aggSignLockHash(_ context.Context, _ kcTransport, _ cluster.Lock) ([]byte, error) {
-	// TODO(corver): Implement lock hash signing by all DVs and aggregation.
+// aggSignLockHash returns the aggregated multi signature of the lock hash
+// signed by all the distributed validator group private keys.
+func aggSignLockHash(ctx context.Context, tcpNode host.Host, peerIdx int,
```

Comment:
@corverroos please, either one line or each parameter on its own line. I think that is how I always saw it. Isn't a linter for that?

Reply:
at this point this isn't linted or specified anywhere